config/routes.rb

```diff
@@ -208,6 +208,12 @@
 
       resources :statements, only: %i[index show], param: :ecf_id
     end
+
+    namespace :teacher_record_service, path: "teacher-record-service", defaults: { format: :json } do
+      namespace :v1 do
+        resources :qualifications, path: "npq-qualifications", only: %i[show], param: :trn
```

Comment:
Do we need to namespace the `path` with a `npq-` prefix? 

Its a new API against our npq specific host, some of the existing APIs are like it to maintain compatibility with old lead provider code carried over from ECF integration but thats not the case here?